src/Components/DrillPage.js

```diff
@@ -140,9 +199,50 @@ const styles = StyleSheet.create({
     textAlign: 'center',
     fontSize: theme.FONT_SIZE_MEDIUM,
   },
-  favoriteContainer: {
+  infoDrill: {
+    color: theme.COLOR_PRIMARY_LIGHT,
+    paddingHorizontal: 30,
+    fontSize: theme.FONT_SIZE_LARGE,
+  },
+  infoSubWrapper: {
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  gradient: {
+    flex: 1,
+    width: 100,
+    height: 100,
+    borderRadius: 50,
+    justifyContent: 'center',
     alignItems: 'center',
   },
+  animation: {
+    flex: 1,
+  },
+  source: {
+    color: theme.COLOR_PRIMARY_LIGHT,
+    paddingHorizontal: 30,
+    fontSize: theme.FONT_SIZE_SMALL,
+  },
+  descriptionGoalsTitle: {
+    marginBottom: 10,
+    textAlign: 'center',
+    fontSize: theme.FONT_SIZE_MEDIUM,
+    color: theme.COLOR_SECONDARY,
+  },
+  descriptionGoalsText: {
+    color: theme.COLOR_PRIMARY,
+    textAlign: 'center',
+    fontSize: theme.FONT_SIZE_LARGE,
+    fontWeight: 'bold',
+  },
+  lines: {
+    borderBottomColor: '#DCDCDC',
```

Comment:
Colors should be part of the theme.
Maybe use an already existing one that is close, like `COLOR_SECONDARY_LIGHT` (e5e5e5)